quelpa.el

```diff
@@ -1811,7 +1824,7 @@ the `quelpa' command has been run in the current Emacs session."
       (when quelpa-self-upgrade-p
         (quelpa-self-upgrade))
       (setq quelpa-cache
-            (cl-remove-if-not #'package-installed-p quelpa-cache :key #'car))
+            (--remove (not (package-installed-p (car it))) quelpa-cache))
```

Comment:
As mentioned, it would be preferable to not make changes like this unless necessary, especially when it adds a new dependency (even though I love `dash`).